### ISSUE TITLE
Buttons does not have focus indicator in high contrast mode

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#164]
Based on our partner's feedback:

> In High Contrast Mode, Focus not shown when we navigate to any buttons(ex: Previous, Next, Quick Fix etc) on the dialog. This is violation of AVT Checkpoint 1.3f Cascading Style sheets
